File: cw/python/FizzBuzz.py
def fizzbuzz(n):
    fb = []
    for i in range(1, n+1):
        if i % 15 == 0:
            fb.append("FizzBuzz")
        elif i % 5 == 0:
            fb.append("Buzz")
        elif i % 3 == 0:
            fb.append("Fizz")
        else:
            fb.append(i)
    return fb

if __name__ == "__main__":
    print(fizzbuzz(3))

File: lang/elixir/fizzbuzz.exs
defmodule FizzBuzz do
    def fizzbuzz(n) when n > 0 do
        Enum.each(1..n, fn(n) ->
            cond do
                rem(n, 3) == 0 and rem(n, 5) == 0 -> IO.puts("FizzBuzz")
                rem(n, 3) == 0 -> IO.puts("Fizz")
                rem(n, 5) == 0 -> IO.puts("Buzz")
                true -> IO.puts(n)
            end
        end)
    end
end

FizzBuzz.fizzbuzz(20)
